pkg/services/session/storage/persistent/encryption_test.go

```diff
@@ -0,0 +1,27 @@
+package persistent
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTokenStore_Encryption(t *testing.T) {
+	pk, err := keys.NewPrivateKey()
+	require.NoError(t, err)
+
+	ts, err := NewTokenStore(filepath.Join(t.TempDir(), ".storage"), WithEncryptionKey(&pk.PrivateKey))
+	require.NoError(t, err)
+
+	data := []byte("nice encryption, awesome tests")
+
+	encryptedData, err := ts.encrypt(data)
```

Comment:
Compare that encrypted and decrypted data are not the same.

Reply:
sure